nsxt/provider.go

```diff
@@ -317,6 +317,7 @@ func Provider() *schema.Provider {
 			"nsxt_edge_upgrade_group":                        dataSourceNsxtEdgeUpgradeGroup(),
 			"nsxt_host_upgrade_group":                        dataSourceNsxtHostUpgradeGroup(),
 			"nsxt_policy_gateway_interface_realization_info": dataSourceNsxtPolicyGatewayInterfaceRealizationInfo(),
+			"nsxt_upgrade_prepare_ready":                     dataSourceNsxtUpgradePrepareReady(),
```

Comment:
Just a naming comment - `prepare_ready` sounds non-optimal to me, maybe `prepare_done`? Other suggestions? @GraysonWu @salv-orlando

Reply:
prepare_complete? I'm ok with both

Reply:
Maybe just `nsxt_upgrade_ready`?